app/serializers/auction_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class AuctionSerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :created_at, :ends_at, :user_id, :starting_price, :bid
```

Comment:
https://github.com/rails-api/active_model_serializers/blob/v0.10.6/docs/general/serializers.md#associations

Try using `has_one` for the bid instead of passing it as an attribute

Reply:
yes! this is what I needed to get it to use the Bid serializer